_includes/modern-appendix.md

```diff
@@ -656,6 +656,13 @@ This is generally sent in response to channel modes, such as a channel being [mo
 
 Indicates that the `JOIN` command failed because the client has joined their maximum number of channels. The text used in the last param of this message may vary.
 
+### `ERR_NOORIGIN (409)`
+
+      ":No origin specified"
```

Comment:
I think this is missing `<client>`. It's not in the original RFC (where it's implicit) but it's always mentioned in modern-irc.